alf/environments/metadrive/geometry.py

```diff
@@ -56,6 +56,12 @@ def __init__(self,
     def bbox(self):
         return self._bbox
 
+    def test(self, points: np.ndarray):
```

Comment:
doc-string and better name such as "within"?

Reply:
Fixed. Also renamed the function as suggested.